azurelinuxagent/common/osutil/default.py

```diff
@@ -554,7 +554,12 @@ def set_selinux_context(self, path, con): # pylint: disable=R1710
             if not os.path.exists(path):
                 logger.error("Path does not exist: {0}".format(path))
                 return 1
-            return shellutil.run('chcon ' + con + ' ' + path)
+            
+            try:
+                shellutil.run_command(['chcon', con, path])
+            except shellutil.CommandError as cmd_err:
```

Comment:
shellutil logs errors by default... we would need to log here to match the original code